x/gov/types/v1/codec.go

```diff
@@ -30,6 +32,17 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	msgservice.RegisterMsgServiceDesc(registry, &_Msg_serviceDesc)
 }
 
+var (
+	amino     = codec.NewLegacyAmino()
+	ModuleCdc = codec.NewAminoCodec(amino)
+)
+
 func init() {
-	RegisterLegacyAminoCodec(legacy.Cdc)
+	RegisterLegacyAminoCodec(amino)
```

Comment:
gov@v1 uses gov@v1beta1. To solve the failing test, I believe we need to add:
```
v1beta1.RegisterLegacyAminoCodec(amino)
```

Reply:
Thanks. done